script.js

```diff
@@ -1,8 +1,10 @@
+
+class Main {
```

Comment:
this is good.

the class definition exists but its never instantiated with `new`